parlai/crowdsourcing/tasks/model_chat/analysis/compile_results.py

```diff
@@ -137,262 +109,255 @@ def compile_results(self) -> pd.DataFrame:
 
         conversation_idx = 0
         conversation_dfs = []
-        for read_folder in read_folders:
-            read_folder_name = os.path.split(read_folder)[-1]
-            for file_name in sorted(os.listdir(read_folder)):
-                if file_name in self.hit_block_list:
```

Comment:
ooof - since we're not using `self.hit_block_list` anymore in this script, I think we should remove the `--hit-block-list` flag from this script and move it into `parlai/crowdsourcing/projects/multisession_chat/human_eval/compile_results.py`, which has the only subclass of this script `ModelChatResultsCompiler` that will still need to use `self.hit_block_list`